nixie/unittests/test_render_diagram.py

```
@@ -53,12 +54,13 @@ async def test_render_diagram_raises_on_failure(
     path = Path("doc.md")
     block = "A-->B"
 
-    async def fake_create_subprocess_exec(*cmd: str, **kwargs: object) -> object:
+    async def fake_create_subprocess_exec(*cmd: str, **_kwargs: object) -> object:
         return object()
 
     async def fake_wait_for_proc(
-        proc: object, path: Path, idx: int, timeout: float
+        _proc: object, _path: Path, _idx: int, _timeout: float
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)